lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -108,6 +108,15 @@
 <!-- Force FCP to be ~5s out, necessary to make sure our render-blocking audits still work -->
 <!-- even when other forces were responsible for delaying the render. -->
 <script src="./fcp-delayer.js?delay=5000"></script>
+
+<!-- FAIL(json-ld): buttons too close together -->
+<script type="application/ld+json">
+{
+  "@context": "http://schema.org",
+  "@type": "Event",
+  "unknownProp": "Oh no"
+}
```

Comment:
is there an expectations change for this?

Reply:
I'm not sure I understand the question - could you elaborate a bit?